decidim-core/app/cells/decidim/card_metadata_cell.rb

```diff
@@ -23,6 +24,68 @@ def initialize(*)
 
     private
 
+    def space_item
+      return unless show_space?
+
+      {
+        text: translated_attribute(participatory_space.title),
```

Comment:
Maybe we need to use `decidim_html_escape(translated_attribute(participatory_space.title))` ?